docs/revup.md

```diff
@@ -96,6 +96,10 @@ Revup is comprised of several sub-commands.
 : Modify a commit in the current stack using the contents of the
 cache. Can also change the commit text.
 
+**revup commit**
+: Create a commit in the current stack using the contents of the
+cache. Can also change the commit text. A convenience wrapper for revup amend --insert
```

Comment:
Not necessary to duplicate this description, just say "A convenience wrapper for revup amend --insert"